RepoMan/RepoMan/Analysis/Scoring/CommentCountScorer.cs

```diff
@@ -22,12 +22,13 @@ public CommentCountScorer(IWordCounter wordCounter)
 
         public override int Count(PullRequest prDetails)
         {
+            var titleCount = _wc.Count(prDetails.Title);
             var bodyCount = _wc.Count(prDetails.Body);
             var commentCount = prDetails.AllComments
                 .Select(c => _wc.Count(c.Text))
                 .Count();
 
-            return bodyCount + commentCount;
+            return titleCount + bodyCount + commentCount;
```

Comment:
Not your error, but probably something I messed up. `CommentCountScorer` should count comments, not words in comments, which is already counted up by the `WordCountScorer`. Hmm. I think? Crap, what is this object meant to do?